tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MetricAppBean.java

```diff
@@ -51,20 +51,45 @@ public class MetricAppBean {
     @Metric(name = "blue")
     private Counter blueCount;
 
+    public static final String NO_TAG_COUNTER = "noTagCounter";
     public static final String TAGGED_COUNTER = "taggedCounter";
+
+    public static final String NO_TAG_HISTOGRAM = "noTagHistogram";
     public static final String TAGGED_HISTOGRAM = "taggedHistogram";
+
+    public static final String NO_TAG_TIMER = "noTagTimer";
     public static final String TAGGED_TIMER = "taggedTimer";
-    public static final String TAGGED_SIMPLETIMER = "taggedSimpleTimer";
-    public static final String TAGGED_METER = "taggedMeter";
+
+    public static final String NO_TAG_GAUGE = "noTagGauge";
     public static final String TAGGED_GAUGE = "taggedGauge";
-    public static final String TAGGED_CONCURRENTGAUGE = "taggedConcurrentGauge";
+
+    public static final String SHARED_METRIC_NAME = "sharedMetricName";
+
+    @Counted(name = SHARED_METRIC_NAME, absolute = true, scope = "customScopeA")
+    public void countMeMetricNameScopeA() {
+
+    }
+
+    @Timed(name = SHARED_METRIC_NAME, absolute = true, scope = "customScopeB")
+    public void timeMeMetricNameScopeB() {
+
+    }
+
+    @org.eclipse.microprofile.metrics.annotation.Gauge(name = SHARED_METRIC_NAME, absolute = true, scope = "customScopeC", unit = "jelly")
```

Comment:
is use of the package name for the Gauge here intentional?

Reply:
Yup, the `Gauge` namespace is used by gauge metric interface so we need to use the fully qualified path when using the annotation.